data/mil2525d/utilities/military-features-utilities/template-gdb-toolbox/scripts/CreateTemplateGDB/UpdateDomains.py

```diff
@@ -0,0 +1,59 @@
+# ----------------------------------------------------------------------------------
```

Comment:
We'll want to try to be consistent (also we probably don't want to apply a bunch of packaging requirements unnecessarily) - I'll just add removing this extra folder to a new issue for future enhancements after the merge - so far the only things I have for this issue are (1) this extra folder issue and (2) checking that the domains exist before adding a field with that domain